crl-test/service_accounts.tf

```diff
@@ -17,9 +23,16 @@ locals {
   ]
 }
 
-resource "google_project_iam_member" "crl" {
+resource "google_project_iam_member" "crl-admin" {
   count = length(local.roles)
   project = google_project.project.name
   role = local.roles[count.index]
-  member = "serviceAccount:${google_service_account.crl.email}"
+  member = "serviceAccount:${google_service_account.crl-admin.email}"
 }
+
+resource "google_project_iam_member" "crl-user" {
+  count = length(local.roles)
```

Comment:
do we need to give the user the same roles? Does the user need any roles at all?

Reply:
For now the scenario I feel we need this user is:
admin create a resource, change acl to the user. 
If that is the case, seems we don't need any of them?